testing/resources/containers/cb-with-exporter.Dockerfile

```diff
@@ -1,18 +1,19 @@
-# Change
 ARG couchbase_server_image=couchbase/server:enterprise-6.6.3
+ARG exporter_version=master
 
 # Build the exporter
 FROM golang:1.17.2 as go_build
 RUN git clone https://github.com/couchbase/couchbase-exporter.git /opt/couchbase-exporter
 
 WORKDIR /opt/couchbase-exporter
+RUN git checkout $exporter_version
```

Comment:
Does this pass Hadolint? It shouldn't... as the RUN lines should be combined. There's no point having them separate, either it'll work or the cache will be invalidated anyway on a new commit.

Reply:
We might have to switch to a script to find Dockerfiles and run Hadolint on each.

Reply:
Yup, it doesn't. Will fix, run hadolint manually, and file a CMOS to extend the linting.